packages/mobile/src/web3/saga.ts

```diff
@@ -263,6 +263,8 @@ export function* getAccount() {
   }
 }
 
+export const getAccount = getWalletAddress
```

Comment:
Can you add a comment here to explain that this is incorrect but requires a larger refactor to fix?